packaging/flavorgen/flavors/util/helpers.go

```diff
@@ -79,7 +79,6 @@ var (
 		regexVar(env.VSphereServerVar),
 		regexVar(env.VSphereTemplateVar),
 		regexVar(env.VSphereStoragePolicyVar),
-		// TODO: Why was thumbprint not here?
```

Comment:
Because it did not necessarily need the escape thing.

Reply:
I don't even understand what "not here" means, do you? (as it is in l.82)